pyproject.toml

```diff
@@ -48,6 +48,9 @@ dependencies = [
     "types-PyYAML",
     "types-tqdm",
     "zstandard",
+    "transformers[torch]",
+    "peft",
```

Comment:
is this used?

Reply:
`peft` is a hidden dependency of our Ultravox model.
Admittedly, it does feel weird to add it here. I might remove it depending on how we integrate the two repos.